lib/ansible/module_utils/spbm_client.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/python
```

Comment:
could you please rename this file to something `vmware_spbm.py` or something.

Reply:
👍